x/emissions/types/keys.go

```diff
@@ -86,4 +86,8 @@ var (
 	PreviousTopicQuantileInfererScoreEmaKey           = collections.NewPrefix(71)
 	PreviousTopicQuantileForecasterScoreEmaKey        = collections.NewPrefix(72)
 	PreviousTopicQuantileReputerScoreEmaKey           = collections.NewPrefix(73)
+	QualifiedInferersKey                              = collections.NewPrefix(74)
```

Comment:
I think the name should be "Active", not "Qualified", throughout the codebase, to keep consistency.